test/unit/find_each_mocker_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'find_each_mocker'
```

Comment:
Moving from lib to app/lib implies not having to require. So I suggest:
1. Remove this line
2. `mv lib/find_each_mocker.rb app/lib/find_each_mocker.rb`

Because `FindEachMocher` is a very simple file when it comes to dependencies and will work this way without having to move or change anything else.